bindings/nodejs/examples/how_tos/evm/send_base_token_to_evm.ts

```diff
@@ -0,0 +1,113 @@
+// Copyright 2021-2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    Client,
+    initLogger,
+    Utils,
+    UnlockCondition,
+    Ed25519Address,
+    SenderFeature,
+    MetadataFeature,
+    Wallet,
+    AddressUnlockCondition,
+    AliasAddress,
+} from '@iota/sdk';
+import {
+    prepareMetadata
+} from './utils';
+
+require('dotenv').config({ path: '.env' });
+
+// Run with command:
+// yarn run-example ./how_tos/outputs/unlock-conditions.ts
+
+const amount = 1000000;
+const gas = 10000;
+const toEVMAddress = '0x48e28C1681BBb92a2E5874113bc740cC11A0FD7a';
+const chainAddress = 'rms1pr75wa5xuepg2hew44vnr28wz5h6n6x99zptk2g68sp2wuu2karywgrztx3';
+
+// Build ouputs with all unlock conditions
+async function run() {
+    initLogger();
+
+    const client = new Client({});
+
+    if (!process.env.STRONGHOLD_PASSWORD) {
+        throw new Error(
+            '.env STRONGHOLD_PASSWORD is undefined, see .env.example',
+        );
+    }
+
+    const wallet = new Wallet({
+        storagePath: process.env.WALLET_DB_PATH,
+    });
+
+    const account = await wallet.getAccount('Alice');
+
+    // Sync new outputs from the node.
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const _syncBalance = await account.sync();
+
+    // After syncing the balance can also be computed with the local data
+    const balance = await account.getBalance();
+    console.log('Balance::', balance.baseCoin);
+
+    const bigAmount: bigint = BigInt(amount);
+    const bigGas: bigint = BigInt(gas);
+    // console.log('amounts:', bigAmount, bigGas);
```

Comment:
Remove all the commented console logs if they should not actually be enabled in this example